### ISSUE TITLE
Please provide examples for RNDIS device

### ISSUE THREAD

**Issue by sidd-kishan** [whitequark/libfx2#14]
It help to build network adpater and usb-ethernet or usb-wifi bridges with the help of this library.

**Comment by whitequark** [comment]
At the moment I have no plans to develop such an example. If you would like assistance I'm happy to be contracted to develop it for you.

**Comment by sidd-kishan** [comment]
Yes, Please let me know how may I contact you and discuss further on the same.

**Comment by whitequark** [comment]
Email me on whitequark@whitequark.org.
